contracts/LpSugar.vy

```diff
@@ -363,20 +516,51 @@ def _byData(_data: address[3], _account: address) -> Lp:
   if gauge_alive:
     emissions = gauge.rewardRate()
 
+  if _account != empty(address):
+    acc_balance = pool.balanceOf(_account)
+    claimable0 = pool.claimable0(_account)
+    claimable1 = pool.claimable1(_account)
+    claimable_delta0: uint256 = pool.index0() - pool.supplyIndex0(_account)
+    claimable_delta1: uint256 = pool.index1() - pool.supplyIndex1(_account)
+
+    if claimable_delta0 > 0:
+      claimable0 += (acc_balance * claimable_delta0) / 10**convert(decimals, uint256)
+    if claimable_delta1 > 0:
+      claimable1 += (acc_balance * claimable_delta1) / 10**convert(decimals, uint256)
+
+  positions: DynArray[Position, MAX_POSITIONS] = empty(DynArray[Position, MAX_POSITIONS])
+
+  positions.append(
```

Comment:
Was testing this and we should return empty positions if there's no account liqudity/rewards.

Lets consider adding a conditional before adding positions
```
if acc_balance > 0 or acc_staked > 0 or earned > 0 or claimable0 > 0:
```